src/python_testing/TC_IDM_1_4.py

```diff
@@ -132,7 +132,7 @@ async def test_TC_IDM_1_4(self):
             asserts.assert_equal(e.status, Status.InvalidAction,
                                  "DUT sent back an unexpected error, we were expecting InvalidAction")
             logging.info("DUT successfully failed to process `MaxPathsPerInvoke + 1` InvokeRequests")
-        except ChipStackError as e:
+        except ChipStackError as e:  # chipstack-ok: This disables ChipStackError linter check
```

Comment:
Same here: also explain WHY we do this. It seems we should do the `With` and then assert on e.err